src/frontend/utils/validators.ts

```diff
@@ -40,9 +40,10 @@ export const erGyldigMånedDato = (felt: IFelt<IPersonBeregning>): IFelt<IPerson
 };
 
 export const erPeriodeGyldig = (felt: IFelt<IPeriode>): IFelt<IPeriode> => {
-    return stringToMoment(felt.verdi.fom, TIDENES_MORGEN).isBefore(
-        stringToMoment(felt.verdi.tom, TIDENES_ENDE)
-    )
+    return moment(felt.verdi.fom).isValid() &&
```

Comment:
Why not check validation of tom?
`moment(felt.verdi.fom).isValid() && moment(felt.verdi.tom).isValid()`

Reply:
Because tom can be null.